src/renderer/src/routes/pages/Send/SelectToken.tsx

```diff
@@ -35,32 +35,45 @@ export const SelectToken = ({ selectedAccount, selectedToken, onSelectToken, act
             {t('tokenToSend')}
           </span>
         </div>
-        <button
+        <Button
           className="flex items-center"
-          disabled={!selectedAccount}
           onClick={modalNavigateWrapper('select-token', {
             state: {
               selectedAccount: selectedAccount,
               onSelectToken: onSelectToken,
             },
           })}
-        >
-          <span className={StyleHelper.mergeStyles('mr-3', textColor && [textColor])}>
-            {selectedToken ? (
-              <div className="flex">
-                <div className="rounded-lg bg-gray-300 w-4.5 h-4.5 flex justify-center items-center">
-                  <BlockchainIcon blockchain={selectedToken.blockchain} type="white" className="w-2.5 h-2.5" />
+          clickableProps={{ className: selectedAccount ? 'hover:bg-gray-300/15 hover:rounded pr-1' : 'pr-1' }}
+          variant="text"
+          colorSchema={active ? 'neon' : 'white'}
+          label=""
+          disabled={selectedAccount ? false : true}
+          labelComponent={
```

Comment:
You can pass a prop called leftIcon and label instead, it will create the same behavior of this labelComponent instead.
```suggestion
         ...otherProps
         label={selectedToken ? StringHelper.truncateString(selectedToken.token.symbol, 4) : t('selectToken')}
         leftIcon={
            selectedToken ? (
              <div className="rounded-lg bg-gray-300 w-4.5 h-4.5 flex justify-center items-center">
                <BlockchainIcon blockchain={selectedToken.blockchain} type="white" className="w-2.5 h-2.5" />
              </div>
            ) : undefined
          }
```